backend/routes/taskRoutes.js

```diff
@@ -10,4 +16,30 @@ const router = express.Router();
 // @acess Private
 router.post("/", [authenticateToken, createTask]);
 
-module.exports = router;
\ No newline at end of file
+// @desc     Fetch all tasks
+// @route    GET /api/tasks/
+// @access   Private
+router.get("/", [authenticateToken, getTasks]);
+
+// @desc     Fetch single task
+// @route    GET /api/tasks/task/:id
+// @acesss   Private
+router.get("/task/:id", [authenticateToken, getTasksById]);
+
+// @desc     Fetch task with day query
+// @route    GET /api/tasks/day/:day
+// @access   Private
+router.get("/day/:day", [authenticateToken, getTasksByDay]);
+
+
+// @desc     Update isStarted state to true
+// @route    PUT /api/tasks/startTask/:id
+// @access   Private
+router.put("/startTask/:id", [authenticateToken, startTask]);
```

Comment:
For consistency, I would have this as `/tasks/:id/start` and `/tasks/:id/end`

Reply:
From my previous comment (about merging the two functions together), it might even be better to merge the tow and keep it as `/tasks/:id/toggle`

Reply:
Fixed: Changes are reflected in recent commits